index.js

```diff
@@ -12,12 +12,14 @@
  * governing permissions and limitations under the License.
  */
 
-const CLI = require('./src/cli.js');
-const ConfigUtils = require('./src/config/config-utils.js');
-require('dotenv').config();
+import CLI from './src/cli.js';
+import { checkNodeVersion, validateDotEnv } from './src/config/config-utils.js';
+import { config } from 'dotenv';
+
+config();
 
 (async () => {
-  await ConfigUtils.checkNodeVersion();
-  await ConfigUtils.validateDotEnv();
-  new CLI().run(process.argv.slice(2));
+  await checkNodeVersion();
+  await validateDotEnv();
+  await new CLI().run(process.argv.slice(2));
```

Comment:
So, this _should_ have been awaited already before?

Reply:
yes